images/primers/docs/components/template.js

```diff
@@ -415,8 +415,8 @@ export function defineDOMSurface({width, height, useChromestage}) {
             method: "POST",
             headers: {'Content-Type': "application/json"},
             body: JSON.stringify({
-              Command: command,
-              Parameters: parameters,
+              command,
+              parameters,
```

Comment:
Is there something I should do to test this change? This was the only place I saw in here that was calling the Chromestage API.

Reply:
Loading /primers/A2/ should test it, I think